Integrations/integration-SplunkPy.yml

```diff
@@ -138,6 +138,31 @@ script:
         if len(service.jobs) >= 0:
             demisto.results('ok')
         sys.exit(0)
+    if demisto.command() == 'splunk-search':
+        t = datetime.utcnow() - timedelta(days=7)
+        time = t.strftime("%Y-%m-%dT%H:%M:%S")
+        kwargs_oneshot = {"earliest_time":time}
+        if demisto.get(demisto.args(), 'earliest_time'):
+            kwargs_oneshot['earliest_time'] = demisto.args()['earliest_time']
+        if demisto.get(demisto.args(), 'latest_time'):
+            kwargs_oneshot['latest_time'] = demisto.args()['latest_time']
+        if demisto.get(demisto.args(), 'event_limit'):
+            kwargs_oneshot['count'] = int(demisto.args()['event_limit'])
+        searchquery_oneshot = demisto.args()['query']
+        if not searchquery_oneshot.startswith('search'):
+            searchquery_oneshot = 'search '+searchquery_oneshot
+        oneshotsearch_results = service.jobs.oneshot(searchquery_oneshot, **kwargs_oneshot)
+
+        reader = results.ResultsReader(oneshotsearch_results)
+        res = []
+        for item in reader:
+            if isinstance(item, results.Message):
+                demisto.results({"Type": 1, "ContentsFormat": "json", "Contents": json.dumps(result.message)})
```

Comment:
`json.dumps(result.message)` => `json.dumps(results.message)`

Reply:
results is a lib param, it's ok